bridge/client/app/_components/ktb-project-settings/ktb-project-settings.component.ts

```diff
@@ -66,8 +67,11 @@ export class KtbProjectSettingsComponent implements OnInit, OnDestroy, PendingCh
 
   readonly state$: Observable<ProjectSettingsState> = this.dataService.keptnInfo.pipe(
     filter((keptnInfo: KeptnInfo | undefined): keptnInfo is KeptnInfo => !!keptnInfo),
-    map((keptnInfo: KeptnInfo) => ({ gitUpstreamRequired: !keptnInfo.metadata.automaticprovisioning })),
-    startWith({ gitUpstreamRequired: undefined })
+    map((keptnInfo: KeptnInfo) => ({
+      gitUpstreamRequired: !keptnInfo.metadata.automaticprovisioning,
+      automaticProvisioningMessage: keptnInfo.bridgeInfo.automaticProvisioningMsg?.trim(),
```

Comment:
Should we move the `trim()` to the bridge server? Then it's one time on bridge server startup instead of every time the data is loaded in the frontend
```ts
const automaticProvisioningMsg = process.env.AUTOMATIC_PROVISIONING_MSG?.trim();
```